examples/quirrel/app/routes/index.tsx

```diff
@@ -0,0 +1,41 @@
+import type { LoaderFunction } from "remix";
+
+import greetingsQueue from "~/queues/greetings.server";
+
+export const loader: LoaderFunction = async () => {
+	await greetingsQueue.enqueue("Groot");
```

Comment:
Could you add some comments here to describe a bit what a real-world use case for doing this in a loader would be?

Reply:
I would say the better real-world use case would be in an action. For instance, a Stripe webhook handler would queue a job to properly handle its payload and immediately return a 200.

Reply:
If you'd like to update this example to do something more like that I think that would be good. You can still do console.log but maybe describe what you would typically use this for with a code comment. Thanks for taking the time :)